region/crateria/west.json

```diff
@@ -414,6 +477,48 @@
                   "requires": []
                 }
               ]
+            },
+            {
+              "id": 9,
+              "strats": [
+                {
+                  "name": "Xray Climb",
+                  "notable": false,
+                  "requires": [
+                    "canLeftFacingDoorXRayClimb",
+                    {"resetRoom": {
+                       "nodes": [4], 
+                       "mustStayPut": true
+                     }},
+                    {"adjacentRunway": {
+                       "fromNode": 4,
+                       "usedTiles": 0.5,
+                       "overrideRunwayRequirements": true,
+                       "useFrames": 200
+                     }},
+                     {"or": [
+                      { "enemyDamage": { "enemy": "Green Space Pirate (standing)", "hits": 1, "type": "laser" } },
```

Comment:
In both of these, is there really only 1 space pirate that attacks?

Reply:
It's only the lowest pirate who can attack.  The others are off camera and inactive.